source/sass/main.scss

```diff
@@ -2,11 +2,6 @@
 
 @import "./mofo-bootstrap/mofo-bootstrap";
 
-// Bootstrap
```

Comment:
we are already importing these in `source/sass/mofo-bootstrap/mofo-bootstrap.scss`